### PR TITLE
Time first request subject email

### PR DESCRIPTION
**Pull request by burnout87**

### CODE REVIEW

**Comment by volodymyrss** [comment]
By the way, this is still hanging

**Comment by burnout87** [comment]
> By the way, this is still hanging

But it was completed, just needs the review

**Comment by volodymyrss** [comment]
Needs to be rebased, I just did it. Let's see the tests

File: tests/test_job_management.py [hunk 1]
```diff
@@ -1072,7 +1075,21 @@ def test_email_submitted_same_job(dispatcher_live_fixture, dispatcher_local_mail
     
     dispatcher_job_state.assert_email(state="submitted", number=2)
     dispatcher_local_mail_server.assert_email_number(2)
-    
+
+    # check the time in the title and filename is still the one of the first request
+    # check the time in the title and filename is still the one of the first request
```
Comment (volodymyrss) [inline tests/test_job_management.py:1080]: duplicate

File: tests/test_job_management.py [hunk 2]
```diff
@@ -1072,7 +1075,21 @@ def test_email_submitted_same_job(dispatcher_live_fixture, dispatcher_local_mail
...
+    for msg in dispatcher_local_mail_server.local_smtp_output:
+        msg_data = email.message_from_string(msg['data'])
+        assert msg_data[
+                   'Subject'] == f"[ODA][submitted] dummy first requested at {time_request_str} job_id: {dispatcher_job_state.job_id[:8]}"
```
Comment (volodymyrss) [inline tests/test_job_management.py:1084]:
```suggestion
                   'Subject'] == f"[ODA][submitted] dummy requested at {time_request_str} job_id: {dispatcher_job_state.job_id[:8]}"
```
user expects it's first requested anyway, I think

**Comment by burnout87** [comment]
Tests now pass, can be merged

**Comment by volodymyrss** [comment]
> Tests now pass, can be merged

You mean can be reviewed of course!

**Review by volodymyrss — APPROVED** [review]

**Comment by volodymyrss** [comment]
I do not really remember why it was not followed, I wanted to look into it deeper but it does not look dangerous. 
(but then, it's #404 so it's not surprising it's getting lost :) )

**Pull request merged**